releasenotes/notes/fix-identity-simplification-no-target-62cd8614044a0fe9.yaml

```diff
@@ -0,0 +1,14 @@
+---
+fixes:
+  - |
+    Fixed an issue with the :func:`~.transpile` function when run with
+    ``optimization_level`` set to ``1``, ``2``, or ``3`` and no
+    ``backend``, ``basis_gates``, or ``target`` argument specified and
+    the input circuit has runs of single qubit gates which can be simplified,
+    or with ``optimization_level=3`` any 2 qubit blocks which are equivalent
+    to an identity matrix the output circuit would not be as optimized as
+    possible. This could have been correct previously by specifying either
+    the ``backend``, ``basis_gates``, or ``target`` arguments on the
+    :func:`~.transpile` call, but now the output will be as simplified as it
+    can without knowing the target gates allowed.
+    Fixed `#9217 <https://github.com/Qiskit/qiskit-terra/issues/9217>`__
```

Comment:
The sentence describing the bug set up is pretty hard to read - it may be clearer to describe it in two separate fix blocks, since to a user, they're fairly distinct anyway, I think.

Reply:
Foxed in: https://github.com/Qiskit/qiskit-terra/pull/9222/commits/1c002fde316303b04b8fe4beab53f8a2e8dd6eac